src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -2232,6 +2232,8 @@ void RadioSpinel<InterfaceType, ProcessContextType>::HandleRcpTimeout(void)
 {
 #if OPENTHREAD_SPINEL_CONFIG_RCP_RESTORATION_MAX_COUNT > 0
     mRcpFailed = true;
+#elif OPENTHREAD_SPINEL_CONFIG_RCP_CUSTOM_RESTORATION
```

Comment:
Do we also need to handle `OPENTHREAD_SPINEL_CONFIG_RCP_CUSTOM_RESTORATION` in `HandleRcpUnexpectedReset` ?